firstvoices/backend/tests/test_apis/test_import_job_api.py

```diff
@@ -395,21 +437,90 @@ def test_confirm_action_404_unknown_key(self):
 
         assert response.status_code == 404
 
-    def test_failed_rows_csv_field_exists(self):
-        site = self.create_site_with_app_admin(Visibility.PUBLIC)
+
+class TestImportJobValidateAction(FormDataMixin, BaseApiTest):
+    API_LIST_VIEW = "api:importjob-list"
+    API_VALIDATE_ACTION = "api:importjob-validate"
+
+    def create_minimal_instance(self, visibility):
+        # Not required for this endpoint
+        return {}
+
+    def get_expected_response(self, instance):
+        # Not required for this endpoint
+        return {}
+
+    def get_list_endpoint(self, site_slug=None, query_kwargs=None):
+        """
+        query_kwargs accept query parameters e.g. query_kwargs={"contains": "WORD"}
+        """
+        url = reverse(self.API_LIST_VIEW, current_app=self.APP_NAME, args=[site_slug])
+        if query_kwargs:
+            return f"{url}?{urlencode(query_kwargs)}"
+        return url
+
+    def setup_method(self):
+        super().setup_method()
+
+        user = factories.UserFactory.create()
+        factories.AppMembershipFactory.create(user=user, role=AppRole.SUPERADMIN)
+
+        self.client.force_authenticate(user=user)
+        self.site = factories.SiteFactory.create(visibility=Visibility.PUBLIC)
 
         data = {
             "title": "Test Title",
-            "data": get_sample_file(
-                "import_job/invalid_dictionary_entries.csv", "text/csv"
-            ),
+            "data": get_sample_file("import_job/minimal.csv", "text/csv"),
         }
 
+        # Initial run
         response = self.client.post(
-            self.get_list_endpoint(site_slug=site.slug),
+            self.get_list_endpoint(site_slug=self.site.slug),
             data=self.format_upload_data(data),
             content_type=self.content_type,
         )
+        assert response.status_code == 201
         response_data = json.loads(response.content)
 
-        assert "failedRowsCsv" in response_data
+        self.import_job = ImportJob.objects.filter(id=response_data["id"])[0]
+
+    @pytest.mark.django_db(transaction=True)
+    def test_exception_fetching_previous_report(self, caplog):
```

Comment:
[minor] I believe you should also assert that the self.import_job.validation_status = JobStatus.FAILED in this test